core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -32,10 +33,14 @@ var TopicsAndSkillsDashboardPage = function() {
     by.css('.protractor-test-create-skill-button'));
   var createSkillButtonSecondary = element(
     by.css('.protractor-test-create-skill-button-circle'));
+  var createSkillButtonSecondaryMobile = element(
+    by.css('.protractor-test-create-skill-button-secondary-mobile'));
```

Comment:
```suggestion
    by.css('.protractor-test-mobile-create-skill-button-secondary'));
```
Can we not have something like this so as to have the class names at one side and the rest at the other side? Ditto eleswhere.

Reply:
Done